addons/website/static/tests/tours/snippet_images_wall.js

```diff
@@ -0,0 +1,46 @@
+odoo.define("website.tour.snippet_images_wall", function (require) {
```

Comment:
You should use the new way `/** @odoo-module **/`

Reply:
Oops, I had noticed it, but then I somehow thought I was in 14.0. Doing it now.